tests/test_plugin_karma.py

```diff
@@ -79,6 +79,12 @@ def test_chained(self):
         self.assertEquals(self.bot.db[_format_key('foo')]['down'], 1)
         self.assertEquals(self.bot.db[_format_key('foo--')]['up'], 1)
 
+    def test_chained_brackets(self):
+        self.bot.dispatch(':root@localhost PRIVMSG #chan :(foo--)++')
```

Comment:
That also seems like unwanted behaviour... I'm modifying the test to have it fail on this behaviour.